indicator-nara/032-berekening-pdf.Rmd

```diff
@@ -0,0 +1,31 @@
+## De droogte-indicator en haar betrouwbaarheid {#h4-2:droogte-indicator}
+
+### Droogte-indicator {#h4-2-1:droogte-indicator}
```

Comment:
Drop the section without the confidence intervals. 
```
\begin{sermon}
Thou shalt not publish uncertain estimates without their confidence interval.
\end{sermon}
```

Reply:
;-)

Reply:
Done